src/client/components/timer.jsx

```diff
@@ -1,21 +1,29 @@
 /* eslint-disable no-shadow */
 import React, { Component } from 'react';
 import { Box, Text } from '@chakra-ui/core';
+import { connect } from 'react-redux';
+import { addScore } from '../store/thunks/gameThunks';
 
-export default class Timer extends Component {
+class Timer extends Component {
```

Comment:
glad it is working - we will need to refactor to functional component for consistency